pkgs/development/python-modules/tensorflow/prefetcher.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-version="2.9.1"
+version="2.9.3"
```

Comment:
I'm surprised this still works. I thought, this version should match the source build one one through mutual dependency on Tensorboard.

Reply:
@veprbl
Good question. As said, I am neither a tensorflow user, nor one of the tensorflow maintainers so I don't fully understand the structure with the given constraints.

Which version should this match? Should version in prefetched.sh == version in binary-hashes.nix?

My guess is that it probably works due to this pr https://github.com/NixOS/nixpkgs/pull/198321 which relaxes version constraints.

The current build succeeds on my aarch64-darwin machine.

Reply:
@tricktron Your explanation makes sense.

Reply:
@veprbl I guess, we could update the prefetcher.sh and tensorboard to 2.10.1. My build just takes 8-9 hours until we know more.

Reply:
@veprbl Nevermind, these are for the binary build of tensorflow which is unrelated to the source build we are trying to fix here.